src/bin/ipa_bench/gen_events.rs

```diff
@@ -34,99 +37,47 @@ pub struct EventBase {
     pub timestamp: Number,
 
     /// 0 (false) if this is a source event. 1 (true) otherwise.
-    pub is_trigger: bool,
-}
+    pub is_trigger_report: bool,
```

Comment:
I wonder if source/trigger events can be better represented as enum

```
  enum Event {
     Source { timestamp, ... },
     Trigger { timestamp, value }
  }
```

code that serializes this enum should make sure both events have the same size on disk/network but at least in the code we will never be able to create a source event with a non-null (0) value